camlibs/lumix/lumix.c

```diff
@@ -36,10 +36,15 @@
 #include <libxml/xmlreader.h>
 
 
-#include <sys/socket.h>
```

Comment:
The lumix driver needs ifdef for the socket includes like the other drivers use. I don't have lumix, thus I couldn't check the driver, but it now compiles at least.

Reply:
Appears sensible.